requirements/edx/base.txt

```diff
@@ -58,7 +58,7 @@ mongoengine==0.7.10
 networkx==1.7
 nltk==2.0.4
 nose==1.3.3
-oauthlib==0.6.3
+oauthlib==0.7.2
```

Comment:
This may have implications on our SSO work. It may be better to submit a separate PR in advance of this one just do to the upgrade to oauthlib. See [this wiki page](https://openedx.atlassian.net/wiki/display/TE/Upgrading+Dependent+Libraries) for our processes on upgrading libraries.